exec/stdiopipe_test.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package exec_test
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"k8s.io/utils/exec"
+)
+
+func ExampleCmd_StderrPipe() {
```

Comment:
What is this?  It isn't called anywhere,and it doesn't explain what it is for

Reply:
https://blog.golang.org/examples

Reply:
It seems the new example does not show up on godoc.org. I will check on that in a separate PR. However, the example still runs as a test case never the less, so it's not completely useless ;)

Reply:
Might be because it's a method of an interface?